### PR TITLE
Kubernetes Ingress support

### PR DESCRIPTION
**Pull request by ahma**
This pull request adds support for using the Kubernetes Ingress controller.

### CODE REVIEW

File: fn/values.yaml [hunk 1]
```diff
@@ -45,9 +111,88 @@ flow:
...
+      #     - flow.local
+
   resources: {}
 
 
+flowui:
+  ## enable or disable Flow UI
+  enabled: true
+
+  ## Number of flow-iu replicas
```
Comment (derekschultz) [inline fn/values.yaml]: s/flow-iu/flow-ui

File: fn/templates/fnui-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          image: "{{ .Values.fnui.image.repository }}:{{ .Values.fnui.image.tag }}"
+          imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
+          ports:
+            - containerPort: {{ .Values.fnui.service.internalPort }}
+          env:
+          - name: PORT
+            value: "{{ default "4000" .Values.fnui.service.internalPort }}"
+          - name: FN_API_URL
+            value: http://{{ template "fullname" . }}-api
+
```
Comment (derekschultz) [inline fn/templates/fnui-deployment.yaml]: remove extra newline?

File: fn/values.yaml [hunk 3]
```diff
@@ -23,19 +12,96 @@ fnserver:
...
+#       - secretName: fn-api-tls
+#         hosts:
+#         - fn-api.local
+
+
+fnui:
+  ## enable or disable Fn UI
+  enabled: true
+
+  ## Number of fn-iu replicas
```
Comment (derekschultz) [inline fn/values.yaml]: s/fn-iu/fn-ui/

**Comment by ahma** [comment]
@derekschultz We have to provide the possibility to enable service ingresses separately. This required because of security and utilization reasons.

**Comment by denismakogon** [comment]
Closing this since ingress controller support was implemented a few commits before.

**Pull request closed without merging**